src/screens/Home.tsx

```diff
@@ -50,6 +31,8 @@ const Home: React.FC = () => {
   const [category, setCategory] = useState("vegetables");
```

Comment:
assim voce conseguiria usar 
```suggestion
  const [category, setCategory] = useState(Categories.Vegetables);
```

Reply:
e parar de usar strings jogadas pq elas podem mudar e voce precisaria mudar elas em todos lugares

Reply:
ok